lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -715,7 +715,7 @@ executionUnitPricesFromPParams pp =
     fromAlonzoPrices prices
   where
     prices = getField @"_prices" pp
```

Comment:
Just curious, do you know why it wouldn't be possible to use `pp ^. #_prices` here?

(Apologies if I'm missing something.)

Reply:
<img width="782" alt="Skärmavbild 2021-11-01 kl  10 08 10" src="https://user-images.githubusercontent.com/304423/139648557-572c3f0b-5703-4dbe-8df4-6563b0e19f6b.png">

Not sure. The `getField` + `HasField` seems more straight-forward than the more magical `^.` + `OverloadedLabels`. I tried some time ago but couldn't get `^.` working.

Reply:
Yeah, I just saw the same error. 🤷🏻‍♂️ 

Oh well, I guess it's something we can save for another day!